en/micro-integrator/docs/develop/create-data-services.md

```diff
@@ -0,0 +1,18 @@
+# Create Data Services Project
```

Comment:
Shall we say 'Configs' instead of Project?

Reply:
+1.